napari/_qt/layers/qt_labels_layer.py

```diff
@@ -288,6 +302,16 @@ def change_preserve_labels(self, state):
         else:
             self.layer.preserve_labels = False
 
+    def change_color_mode(self, text):
+        """Change color mode of label layer.
+
+        Parameters
+        ----------
+        state : QComboBox
+            ComboBox to change between color mode.
+        """
```

Comment:
This docstring is outdated — should be fixed in a followup PR. The argument name 'text' isn't quite descriptive enough... `new_mode`?